client/lib/posts/post-counts-store.js

```diff
@@ -19,30 +19,19 @@ const sum = obj => {
 var emitter = require( 'lib/mixins/emitter' ),
 	PostListStore = require( './post-list-store-factory' )(),
 	PostsStore = require( './posts-store' ),
-	sites = require( 'lib/sites-list' )(),
 	postUtils = require( 'lib/posts/utils' ),
 	Dispatcher = require( 'dispatcher' );
 
 var _counts = {},
 	PostCountsStore;
 
 /**
- * Get a normalized numberic siteId
- * @param {String|Integer} id - A site id (numeric site ID, site.slug, or API $site path fragment
- * @return {Integer} - normalized numeric site id
+ * Get siteId to use
+ * @param {Integer} id - A site id (numeric site ID)
```

Comment:
Hi @gwwar, yes at the time of this PR I had checked for the usages, and now I rechecked again and I really think we always pass a numeric siteId to this function.

The only direct usage of  lib/posts/post-counts-store.js is my-sites/posts/posts-navigation.jsx and it seems to use a  numeric siteId all the time.  The handling of action RECEIVE_POST_COUNTS also seems to use getSiteId and there we also use the numeric id.

Probably before my-sites/posts/posts-navigation.jsx  component was reduxified we used other types of id e.g. site slug but now that does not look the case anymore.